ras_party/controllers/account_controller.py

```diff
@@ -208,13 +211,13 @@ def verify_token(token, session):
         duration = current_app.config["EMAIL_TOKEN_EXPIRY"]
         email_address = decode_email_token(token, duration)
     except SignatureExpired:
-        raise RasError('Expired email verification token', status=409, token=token)
+        raise ClientError('Expired email verification token', status=409, token=token)
     except (BadSignature, BadData) as e:
         raise RasError('Unknown email verification token', status=404, token=token, error=e)
 
     respondent = query_respondent_by_email(email_address, session)
     if not respondent:
-        raise RasError("Respondent does not exist.", status=404)
+        raise RasError("Respondent does not exist.  Unable to verify token.", status=404)
```

Comment:
Why don't all instances of 4XX status codes use ClientError?

Reply:
It depends on the circumstances.  For example, all `Respondent does not exist` errors are 404s.  In some cases, it doesn't matter if the exception is raised so it's a ClientError (e.g. if a user mistypes a respondemt id).  In other cases, it's an issue if the respondent isn't found if we're expecting it to, so RasError.

Reply:
I think by doing this we're choosing which exception to raise based on what log level we want the message output at. This would be better dealt with by the service calling this endpoint to interpret the result and how to log it.

Also, by raising it as a `RasError` it's logged as an exception with the text 'Uncaught exception'.